jupyterhub/jupyterhub_config.py

```diff
@@ -113,8 +113,13 @@
 #===========================================================================
 #                            Other Configuration
 #===========================================================================
-c.Spawner.cpu_limit = 16
-c.Spawner.mem_limit = '40G'
+if "SPAWNER_CPU_LIMIT" in os.environ:
+    c.Spawner.mem_limit = os.environ['SPAWNER_CPU_LIMIT']
+else: raise Exception("SPAWNER_CPU_LIMIT not set in .env file")
+
+if "SPAWNER_MEM_LIMIT" in os.environ:
+    c.Spawner.mem_limit = os.environ['SPAWNER_MEM_LIMIT']
+else: raise Exception("SPAWNER_MEM_LIMIT not set in .env file")
```

Comment:
It would be better to not raise an exception when config parameters (that are not essential) are missing. 
Just use defaults in this case

Reply:
new commit should fix this